src/App.tsx

```diff
@@ -20,7 +20,11 @@ export class App extends Component<{}, State> {
   }
 
   handleKey = (event: KeyboardEvent) => {
-    this.setState({ pressedKey: event.key });
+    if (event.key === ' ') {
```

Comment:
better use `event.code` to check if ti is `Space`

Reply:
and don't forget to deploy your changes
<img width="251" alt="image" src="https://user-images.githubusercontent.com/55802630/183102214-959beadf-ccaa-4b85-84da-4d4ef0831261.png">